mule-gateway/modules/ROOT/pages/mule-gateway-metrics.adoc

```diff
@@ -18,6 +18,19 @@ Even though no sensitive information is collected using this configuration, you
 
 `anypoint.platform.metrics_enabled=false`
 
+== Log analytics event data
+
+In Mule 4, it is possible to enable API analytics events logging by adding the following entry in the `wrapper.conf` file:
+
+`anypoint.platform.log_analytics_events=true`
+
+This will result in `INFO` logs of the following format:
+
+`com.mulesoft.mule.runtime.gw.analytics.logger.AnalyticsHttpEventsLogger: {"api_id":XX,"api_version_id":XX,"org_id":"XXX","host_id":"XXXX","client_id":XXX,"transaction_id":XXX,"received_ts":"XXX","replied_ts":"XXX","client_ip":"XX","verb":"GET","path":"/backend","status_code":200,"user_agent":"XXX","request_bytes":0,"response_bytes":150,"request_disposition":"XXX","policy_violation":null,"event_id":"XXX","deployment_type":null}`
+
+IMPORTANT: This feature might affect the performance of the request processing, so take it in consideration.
```

Comment:
I would clarify that the performance impact is due to the amount of logs that are generated and not because of the feature itself